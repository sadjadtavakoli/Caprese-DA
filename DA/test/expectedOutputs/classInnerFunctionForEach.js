
#1 function StyleContextStack-test/unit_tests/classinnerfunctionforeach.js-1-25 entered from classInnerFunctionForEach.js-test/unit_tests/classinnerfunctionforeach.js-1-34
#20 function auto-test/unit_tests/classinnerfunctionforeach.js-20-23 entered from classInnerFunctionForEach.js-test/unit_tests/classinnerfunctionforeach.js-1-34
#3 function autopush-test/unit_tests/classinnerfunctionforeach.js-3-18 entered from auto-test/unit_tests/classinnerfunctionforeach.js-20-23
#13 function arrowFunction-test/unit_tests/classinnerfunctionforeach.js-13-16 entered from autopush-test/unit_tests/classinnerfunctionforeach.js-3-18
#32 function test-test/unit_tests/classinnerfunctionforeach.js-32-34 entered from arrowFunction-test/unit_tests/classinnerfunctionforeach.js-13-16
#13 function arrowFunction-test/unit_tests/classinnerfunctionforeach.js-13-16 entered from autopush-test/unit_tests/classinnerfunctionforeach.js-3-18
#32 function test-test/unit_tests/classinnerfunctionforeach.js-32-34 entered from arrowFunction-test/unit_tests/classinnerfunctionforeach.js-13-16
#13 function arrowFunction-test/unit_tests/classinnerfunctionforeach.js-13-16 entered from autopush-test/unit_tests/classinnerfunctionforeach.js-3-18
#32 function test-test/unit_tests/classinnerfunctionforeach.js-32-34 entered from arrowFunction-test/unit_tests/classinnerfunctionforeach.js-13-16
#28 function arrowFunction-test/unit_tests/classinnerfunctionforeach.js-28-30 entered from auto-test/unit_tests/classinnerfunctionforeach.js-20-23
#end Execution