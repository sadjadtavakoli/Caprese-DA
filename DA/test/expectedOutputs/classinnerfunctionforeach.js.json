{"StyleContextStack-test/unit_tests/classinnerfunctionforeach.js-1-25":{"impacted":["t_1"]},"auto-test/unit_tests/classinnerfunctionforeach.js-20-23":{"impacted":["t_1","t_12"]},"test/unit_tests/classinnerfunctionforeach.js":{"impacted":["t_7"]},"autopush-test/unit_tests/classinnerfunctionforeach.js-3-18":{"impacted":["t_7","t_200"]},"test-test/unit_tests/classinnerfunctionforeach.js-32-34":{"impacted":["t_200"]},"arrowAnonymousFunction-test/unit_tests/classinnerfunctionforeach.js-28-30":{"impacted":["t_7"]},"keyMap":{"t_1":"test/unit_tests/classinnerfunctionforeach.js","t_5":"StyleContextStack-test/unit_tests/classinnerfunctionforeach.js-1-25","t_7":"auto-test/unit_tests/classinnerfunctionforeach.js-20-23","t_12":"autopush-test/unit_tests/classinnerfunctionforeach.js-3-18","t_200":"arrowAnonymousFunction-test/unit_tests/classinnerfunctionforeach.js-13-16","t_25":"test-test/unit_tests/classinnerfunctionforeach.js-32-34","t_71":"arrowAnonymousFunction-test/unit_tests/classinnerfunctionforeach.js-28-30"}}